projects/project-1/assignment/assignments_new file.md

```diff
@@ -382,25 +381,22 @@ I have only added tables that have errors in them and are not redundant.</p>
 
 - <b>A/A, Ai/Ai, B/B, Bi/Bi</b>: see Exercise 1
 
-- <b>A/Ai</b>: Suppose that A is transitive and Ai is irreflexive. This means that if xAy and yAx hold, it also mean that yAix and xAiy holds. The  it should be the case that xAix. However, Ai i 
-Therefore, it does not hold that A is transitive and Ai is irreflexive.
-
+- <b>A/Ai</b>: Suppose that A is transitive and Ai is irreflexive. This means that if xAy and yAx hold, xAx holds. It also means that xAix should hold. However, Ai is irreflextive so xAix doesn't hold. Therefore, it does not hold that A is transitive and Ai is irreflexive.
 
+- <b>Ai/A</b>: Suppose that Ai is transitive and A is irreflexive. This means that if xAiy and yAix hold, xAix holds. It also means that xAx should hold. However, A is
+irreflextive so xAx doesn't hold. Therefore, it does not hold that Ai is transitive and A is irreflexive.
```

Comment:
See previous comment. 

Also, why are you assuming xAiy and yAix?

Reply:
I will fix them! 😿